src/test/scala/org/virtuslab/beholder/UserMachinesView.scala

```diff
@@ -22,6 +23,8 @@ trait UserMachinesViewComponent
 
   import unicorn.profile.api._
 
+  implicit val dateSchema: Schema[Date] = Schema.string[Date]
```

Comment:
No format here?

Reply:
turns out `Format[Date]` is availabile straight from play-json

Reply:
ah missed that you meant `.format` on `Schema`, added